### DISCUSSION TITLE
Deeplink to a file doesn’t open from Safari

### DISCUSSION THREAD

**Discussion by rjchatfield** [community/community#29151] [Mobile]
What I expect:

- Open https://github.com/gottesmm/swift-evolution/blob/consuming-nonconsuming-pitch-v1/proposals/000b-consuming-nonconsuming.md in Safari
- Top banner allows me to open this link in your iOS App

What actually happens:

- Open https://github.com/gottesmm/swift-evolution/blob/consuming-nonconsuming-pitch-v1/proposals/000b-consuming-nonconsuming.md in Safari
- Top banner suggests I open your app, but does not deep link me to this page. 

I also expect to maybe hop out to the app when I navigated to this link in the first place. I found that link from the Swift Forums, tapping it should have taken me to your app?!

IMHO, reading long proposals like this in the iOS App is better than the Mobile Web (which has too much horizontal padding). 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.75.0 (74400986)
Device: iPhone 13 Pro (iOS 16.0)
</details>

**Comment by alcere** [comment]
Hi @rjchatfield for reporting! I oppened an issue and the team will investigate the problem!
